staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/values_test.go

```diff
@@ -619,3 +619,67 @@ func TestMapper(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkUnstructuredToVal(b *testing.B) {
+	b.ReportAllocs()
+	b.ResetTimer()
```

Comment:
Nit: is `ResetTimer` needed here considering no time consuming operations happen before the benchmark starts running?

Reply:
Though it's only an assignment operation, it never hurt to have ResetTimer^

Reply:
> Though it's only an assignment operation, it never hurt to have ResetTimer^

From a runtime perspective it doesn't hurt.

From a code readability perspective, it begs the reader to wonder why the call is present, and so it a bit of a needless distraction.  Let's remove it.